charts/vsphere-cpi/v1.0.0/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: vsphere-cpi
+version: 1.0.0
+appVersion: 1.0.0
+description: vSphere CPI driver
+sources:
+  - https://github.com/kubernetes/cloud-provider-vsphere
+maintainers:
+  - name: Rancher
+    email: rajashree.28m@gmail.com
```

Comment:
Should this be updated to a rancher email?

Reply:
haha yeah I guess, I didn't know which one to add and I forgot asking this. @PennyScissors what do we usually use?

Reply:
I think you need email/email that resolve to a github account or the tool will fail. If that's the case, you could add your rancher email to github, and then change it to your rancher email.

Reply:
is this required? if not can we just remove this section?

Reply:
As per offline discussion with @cbron adding his email for now (thanks)